yaksh/models.py

```diff
@@ -747,6 +749,8 @@ class Question(models.Model):
     # Check assignment upload based question
     grade_assignment_upload = models.BooleanField(default=False)
 
+    min_time =  models.IntegerField("time in minutes", default=0)
```

Comment:
Yes but did not add sample answer yet because the solution will be in the video, and there can be multiple answers for the question.
Will implement this locally and will push after a brief discussion.